components/layout_2020/query.rs

```diff
@@ -741,7 +741,7 @@ fn rendered_text_collection_steps<'dom>(
                     items.push(InnerOrOuterTextItem::Text(String::from(" ")));
                 };
 
-                if transformed_text.len() > 0 {
+                if !transformed_text.is_empty() == true {
```

Comment:
You shouldn't need to compare a boolean expression against true.

```suggestion
                if !transformed_text.is_empty() {
```

Reply:
Thank I forgot about this.🙃

Reply:
Should be ok now